robottelo/datafactory.py

```diff
@@ -146,7 +146,9 @@ def test_something(value):
     :param argvalues: to be passed to parametrize
     :return: dict
     """
-    return {'argvalues': argvalues, 'ids': map(str, range(len(argvalues)))}
+    return {
+        'argvalues': argvalues, 'ids': list(map(str, range(len(argvalues))))
```

Comment:
Why do we need that `map` here after all? Just to have indexes as strings? We could use something simpler and more descriptive like list generator, e.g.:
```python
'ids': [str(index) for index in range(len(argvalues))]
```

Reply:
Good tip. It's useless to list(map([list]). I'll do it as you suggested.

Generally map is a *lot* faster then list comprehension, I know here we don't chase performance :-)
```
$ python3 -m timeit "map(str, range(1000))"
1000000 loops, best of 3: 0.385 usec per loop

$ python3 -m timeit "[str(index) for index in range(1000)]"
10000 loops, best of 3: 193 usec per loop
```

Reply:
but after converting map to list performance is almost that slow like with list comprehension, heh
```
$ python3 -m timeit "list(map(str, range(1000)))"
10000 loops, best of 3: 152 usec per loop
```
152 vs. 193

Reply:
It's not 500 times faster, mapping is lazy, nothing was actually calculated until you tried to access elements or convert them to a list :)

Reply:
@abalakh You seem to know a lot about snake guts ;-) Thx